README.md

```diff
@@ -397,10 +397,16 @@ render conn, data: data, opts: [meta: meta_data]
 
 ### Key Format (for Attribute, Relationship and Query Param)
 
-By default keys are `dash-erized` as per the jsonapi.org recommendation, but
-keys can be customized via config.
+By default keys are `dash-erized` as per the JSON:API 1.0 recommendation, but keys can be customized via config.
 
-In your `config.exs` file:
+In your `config.exs` file you can use `camelcased` recommended by upcomming JSON:API 1.1:
```

Comment:
From what I can tell "camel case" is two words, so I think this would be better as `camel_cased`.

Reply:
Yep, wasn't sure about it.